fast_api/routes/data_browser.py

```diff
@@ -258,3 +292,18 @@ def update_data_slice(
         handle_error(e, user.id, project_id)
 
     return pack_json_result({"data": {"updateDataSlice": {"ok": ok}}})
+
+
+def handle_error(exception: Exception, user_id: str, project_id: str):
+    general.rollback()
+    if exception.__class__ == TooManyRecordsForStaticSliceException.__class__:
```

Comment:
maybe best with instance of as the class string compare feels off
```suggestion
    if isinstance(exception,TooManyRecordsForStaticSliceException):
```